examples/main/constants.js

```diff
@@ -59,10 +60,6 @@ export const EVENTS = [
 export const INITIAL_OPTIONS = {
   click: true,
   doubletap: true,
-  pointermove: true,
```

Comment:
Why are these removed?

Reply:
These are just default checked boxes in the example. I find them flood the log quite a bit.